.gitignore

```diff
@@ -1,4 +1,5 @@
 .DS_Store
-node_modules/
-lib/
+node_modules
+flow-typed
```

Comment:
Yeah, I expected there to be. 

I was also disappointed that Flow doesn't let you use `[options]` &/or `[include]` instead of specifying `@flow` on every file.